tests/test_sv_sim.py

```
@@ -0,0 +1,350 @@
+    """Fixture to create and return a Qiskit AerSimulator."""
+    return AerSimulator(method="statevector")
+
+
+@pytest.fixture
+def pyqasm_simulator():
+    """Fixture to create and return a pyqasm simulator."""
+    return Simulator(seed=22)
+
+
+@pytest.mark.parametrize(
+    "qasm, description",
```

Ruff (0.15.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)